inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -376,6 +376,45 @@ void FrontEnd::parseLayer(const Model& model, const ie::CNNLayerPtr& layer, cons
     }
 }
 
+void FrontEnd::processTrivialCases(const Model& model) {
+    DataVector trivialInputs, trivialOutputs;
+
+    for (const auto& data : model->datas()) {
+        if (data->usage() == DataUsage::Input || data->usage() == DataUsage::Intermediate || data->usage() == DataUsage::Const) {
+            if (data->numConsumers() == 0 && data->childDataToShapeEdges().size() == 0) {
```

Comment:
Nitpick: is `data->childDataToShapeEdges().empty` available?

Reply:
I've extracted this check to a separate method which uses `empty()`. Thanks!